### PR TITLE
Worked on #15 issue

### PR DESCRIPTION
**Pull request by ItsRoy69**
@nehagup made the necessary changes can you check and merge the commit.

Thank You

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```
Comment (nehagup) [inline CODE_OF_CONDUCT.md]: It seems you have picked this template from "Contributor Covenant. Can you please make the relevant changes for Keploy?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+  <img align="center" src="https://avatars.githubusercontent.com/u/92252339?s=200&v=4" height="20%" width="20%" />
+</p>
+
+<p align="center">
+
+  <a href="CODE_OF_CONDUCT.md" alt="Contributions welcome">
+    <img src="https://img.shields.io/badge/Contributions-Welcome-brightgreen?logo=github" /></a>
+  
+  <a href="https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA" alt="Slack">
+    <img src=".github/slack.svg" /></a>
```
Comment (nehagup) [inline README.md:13]: I checked the "rich text" display/view. It is still not showing the badges. 🤔

**Review by nehagup — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ItsRoy69) [reply CODE_OF_CONDUCT.md]: Made necessary changes and pushed in https://github.com/ItsRoy69/website/blob/main/CODE_OF_CONDUCT.md

If possible checkout and I will make another pull request

File: README.md (hunk 2 above)
Comment (ItsRoy69) [reply README.md:13]: image

This is showing in my current repo https://github.com/ItsRoy69/website 😅..

**Comment by nehagup** [comment]
@ItsRoy69 can you please sign the DCO?

**Comment by ItsRoy69** [comment]
@nehagup made the necessary changes can you look into it please ?

**Review by nehagup — CHANGES_REQUESTED** [review]
The email we publicly refer is hello [at] keploy.io. Rest looks good! 👏

**Review by nehagup — APPROVED** [review]
LGTM!

**Pull request merged**